spec/features/cart_feature_spec.rb

```diff
@@ -17,4 +17,21 @@
     expect(current_path).to eq(cart_path)
     expect(page).to have_content("sour cream")
   end
+
+  it "let's a logged-in user check out" do
+    @user = User.create(full_name: 'B G', email: 'test@mail.com', password: 'password')
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.